variants/aws-ecs-2/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+[package]
+name = "aws-ecs-2"
+version = "0.1.0"
+edition = "2021"
+publish = false
+build = "../build.rs"
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
```

Comment:
nit: it's a bit inconsistent to add this to `aws-ecs-2` but not `aws-ecs-2-nvidia` (or `aws-ecs-1` as well). Doesn't actually matter since none of these have a `README.md`.

Reply:
Oh, I saw that we have the same line for most of the variants. I can clean up all of them in a separate PR.